### ISSUE TITLE
foreign key constraint issue with new elixir DB setup

### ISSUE THREAD

**Issue by akinsey** [epochtalk/epochtalk#280]
`
stack: error: insert or update on table "reports_posts" violates foreign key constraint "status_id_fk"
`

**Comment by akinsey** [comment]
Enums were missing for the `administrations.report_statuses` table

```
/* Insert report statuses */
INSERT INTO administration.reports_statuses (status, priority) VALUES ('Pending', 1);
INSERT INTO administration.reports_statuses (status, priority) VALUES ('Reviewed', 2);
INSERT INTO administration.reports_statuses (status, priority) VALUES ('Ignored', 3);
INSERT INTO administration.reports_statuses (status, priority) VALUES ('Bad Report', 4);
```

**Comment by akinsey** [comment]
@wangbus

**Comment by wangbus** [comment]
Fixed

**Comment by akinsey** [comment]
We need to insert those rows in code somewhere

**Comment by akinsey** [comment]
Resolved in 0ae92b462e9ccdbc7c05a203f795928bd87bff9e
